ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -48,6 +53,26 @@ public enum PillButtonStyle: Int {
             return  Colors.PillButton.Outline.titleSelected
         case .filled:
             return UIColor(light: Colors.primary(for: window), dark: Colors.PillButton.Outline.titleSelected)
+        case .transparent:
+            return UIColor(light: Colors.primary(for: window), dark: Colors.primaryShade10(for: window))
+        }
+    }
+    
+    func font() -> UIFont {
+        switch self {
+        case .outline, .filled:
+            return Fonts.button4
+        case .transparent:
+            return Fonts.button1
+        }
+    }
+    
+    func selectedFont() -> UIFont {
+        switch self {
+        case .outline, .filled:
+            return Fonts.button4
+        case .transparent:
+            return Fonts.preferredFont(forTextStyle: .subheadline, weight: .bold)
         }
```

Comment:
Sorry if this has already been iterated away from based on previous comments, but thoughts on calling this func font(for selected: Bool) and combining the two cases? That could dedupe Fonts.btton4 for outline and filled styles  for a bit more clarity